### PR TITLE
Add Scala.IO as next FLOSS Spree

### PR DESCRIPTION
**Pull request by vil1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,15 +41,25 @@ Here is a list of projects that you could contribute to during the spree:
...
+[scalajs-bundler]: https://github.com/scalacenter/scalajs-bundler
 [endpoints]: https://github.com/julienrf/endpoints
-[endpoints-issues]: https://github.com/julienrf/endpoints/issues?q=is%3Aissue+is%3Aopen+label%3Aspree
-[krever]: https://github.com/Krever
+[fs2-cassandra]: https://github.com/Spinoco/fs2-cassandra
+[fs2-rabbit]: https://github.com/gvolpe/fs2-rabbit
+[fs2-redis]: https://github.com/gvolpe/fs2-redis
+[ZIO]: https://github.com/scalaz/scalaz-zio
 
 Who will be leading the Scala Open Source Spree?
```
Comment (Duhemm) [inline README.md]: I think that this needs to be updated too? @darjutak

File: README.md [hunk 2]
```diff
@@ -41,15 +41,25 @@ Here is a list of projects that you could contribute to during the spree:
...
 | ----------------------------------------------------------------------------------------- | ------------------- |
-| Orkestra (https://github.com/Orkestra-Tech/orkestra)                | @joan38           |
-| [Chimney](https://github.com/scalalandio/chimney/) - Scala library for boilerplate-free data transformations | @krzemin |
-| [julienrf/endpoints][endpoints] - API definition shared between client, server and docs ([issues][endpoints-issues])   | [@Krever][krever]   |
-| [Bloop](https://github.com/scalacenter/bloop) - Scala build server | [@tues](https://github.com/tues) |
+| [Fury]                | @propensive           |
+| [scala.js] and/or [scalac] | @sjrd |
+| [Hamsters] A mini Scala utility library   | @dgouyette   |
+| IntelliJ Scala plugin | @jastice  |
+| [scalajs-bundler], [endpoints], [scalac] | @ulienrf |
```
Comment (Duhemm) [inline README.md]: Typo in Julien's username (the j is missing)

File: README.md [hunk 3]
```diff
@@ -10,15 +10,15 @@ get this awesome t-shirt:
...
-| Time                    | September 9th, 9:00am - 4:00pm                                                               |
-| Location                | Olivia Business Centre, Olivia FOUR, [aleja Grunwaldzka 472a](https://www.google.com/maps/search/?api=1&query=aleja+Grunwaldzka+472a%2C+Gdansk%2C+pl)					       |
-| Conference registration | **Not Required**                                                                           |
-| Event Registration      | **Free, required**: [event-registration](https://www.meetup.com/FunctionalTricity/events/253573703/)                                                  |
-| Food                    | **Free**: Sponsored by Scalac                                                          |
+| Time                    | October 30th, 9:00am - 5:00pm, October 31st 9:00am - 4:00pm                                |
+| Location                | [CPE Lyon](https://www.google.com/maps/search/?api=1&query=aleja+Grunwaldzka+472a%2C+Gdansk%2C+pl) Bâtiment Hubert Curien, Domaine Scientifique de la Doua, 43 Boulevard du 11 Novembre 1918, 69100 Villeurbanne					       |
```
Comment (Duhemm) [inline README.md]: The link needs to be updated too

File: README.md [hunk 4]
```diff
@@ -10,15 +10,15 @@ get this awesome t-shirt:
...
+| Conference registration | **Required** [event registration](https://scala.io)                                        |
```
Comment (Duhemm) [inline README.md]: I wouldn't call this link `event registration` given that it says below that the event registration is not required.

File: README.md (hunk 3 above)
Comment (vil1) [reply README.md]: Good catch Thx

File: README.md (hunk 4 above)
Comment (vil1) [reply README.md]: no the registration to the conference is required to have access to the spree, but no additional registration is needed beyond that

File: README.md (hunk 1 above)
Comment (vil1) [reply README.md]: I think it's correct

File: README.md (hunk 4 above)
Comment (darjutak) [reply README.md]: @vil1 to make it clear: it is limited to the conference participants; for the ones that would like to join only the spree, they need to pay the ticket?
Usually, the sprees are free, sponsored by a company or two... before I go ahead and promote this I need to know, thanks

Comment (vil1) [reply README.md]: @darjutak I confirm, a ticket to the conference will be mandatory.

**Pull request merged**